docs/using-gitbase/security.md

```diff
@@ -0,0 +1,104 @@
+# Security
+
+## User credentials
+
+User credentials can be specified in the command line or in the user file. For a single user this can be done with parameters `--user` and `--password`:
+
+```
+gitbase server --user root --password r00tp4ssword! -d /my/repositories/path
+```
+
+If you want to have more than one user or not having the password in plain text, then the user file can be useful. You can keep passwords, permissions and user names in the following format:
+
+```json
+[
+  {
+    "name": "root",
+    "password": "*2470C0C06DEE42FD1618BB99005ADCA2EC9D1E19",
+    "permissions": ["read", "write"]
+  },
+  {
+    "name": "user",
+    "password": "plain_passw0rd!"
+  }
+]
+```
+
+You can either specify passwords as a plain text or hashed. Hashed version uses the same format as MySQL 5.x passwords. You can generate the native password with this command, remember to prefix the hash with `*`:
+
+```
+echo -n password | openssl sha1 -binary | openssl sha1 | tr '[:lower:]' '[:upper:]'
+```
+
+There are two permissions you can set for users, `read` and `write`. `read` only allows executing read-only queries that do not modify the internal state or the data itself. `write` is needed to create and delete indexes or lock tables. If no permissions are set for a user the default permission is `read`.
+
+Then you can specify which user file to use with the `--user-file` parameter:
+
+```
+gitbase server --user-file /path/to/user-file.json -d /my/repositories/path
+```
+
+## Audit
+
+Gitbase offers audit traces on logs. Right now, we have three different kinds of traces: `authentication`, `authorization` and `query`
```

Comment:
I believe the wording is correct now but if we want to stick to the standard nomenclature this would be:

- audit traces -> audit trails
- trace -> record

https://en.wikipedia.org/wiki/Audit_trail